packages/storybook/stories/ControlBar/Buttons/Participants/Participants.stories.tsx

```diff
@@ -0,0 +1,183 @@
+import { ParticipantsButton } from '@azure/communication-react';
+`;
+
+const getDocs: () => JSX.Element = () => {
+  return (
+    <>
+      <Title>ParticipantsButton</Title>
+      <Description of={ParticipantsButton} />
+      <Description>
+        Note: When overriding a render, like using `onRenderIcon` or `onRenderText`, don not forget to add a unique key
+        to each element to avoid warning for children in a list.
+      </Description>
+
+      <Heading>Importing</Heading>
+      <Source code={importStatement} />
+
+      <Heading>Default Example</Heading>
+      <Description>
+        The default `ParticipantsButton` component shows `People` icon with no label, and contains menu items displaying
+        the number of participants with a list as sub-menu like shown in the example below.
+      </Description>
+      <Canvas mdxSource={ParticipantsButtonDefaultExampleText}>
+        <ParticipantsButtonDefaultExample />
+      </Canvas>
+
+      <Heading>ParticipantsButton with default label</Heading>
+      <Description>
+        You can display the button label which, by default, will show below the icon as `People`.
+      </Description>
+      <Canvas mdxSource={ParticipantsButtonWithLabelExampleText}>
+        <ParticipantsButtonWithLabelExample />
+      </Canvas>
+
+      <Heading>ParticipantsButton with local user excluded from participant list</Heading>
+      <Description>
+        When excluding local user from participant list, `ParticipantsButton` displays their presence to the call as a
+        menu item above the number of remote participants like shown in the example below.
+      </Description>
+      <Canvas mdxSource={ParticipantsButtonWithUserExcludeFromListExampleText}>
+        <ParticipantsButtonWithUserExcludeFromListExample />
+      </Canvas>
+
+      <Heading>ParticipantsButton with copyable call invitation link</Heading>
+      <Description>
+        When opting to have a copyable link available, `ParticipantsButton` displays it as a menu item below the number
+        of participants like shown in the example below.
+      </Description>
+      <Canvas mdxSource={ParticipantsButtonWithCallLinkExampleText}>
+        <ParticipantsButtonWithCallLinkExample />
+      </Canvas>
+
+      <Heading>ParticipantsButton with option to mute all participants</Heading>
+      <Description>
+        When opting to have the possibility of muting all participants, `ParticipantsButton` displays this options as a
+        sub-menu item of the number of participants, just below the list of participants, like shown in the example
+        below.
+      </Description>
+      <Canvas mdxSource={ParticipantsButtonWithMuteAllOptionExampleText}>
+        <ParticipantsButtonWithMuteAllOptionExample />
+      </Canvas>
+
+      <Heading>ParticipantsButton with cutom styles</Heading>
+      <Description>
+        You can change the styles of `ParticipantsButton` through its `styles` props as you would customized any Button
```

Comment:
nit: customized - customize
```suggestion
You can change the styles of `ParticipantsButton` through its `styles` props as you would customize any Button
```